CodeJam.Main/Strings/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Text;
+
+using JetBrains.Annotations;
+
+namespace CodeJam.Strings
+{
+	/// <summary>
+	/// <see cref="StringBuilder"/> class extensions.
+	/// </summary>
+	[PublicAPI]
+	public static class StringBuilderExtensions
+	{
+		/// <summary>
+		/// Determines whether the StringBuilder is null or empty.
+		/// </summary>
+		/// <param name="value">The StringBuilder to test.</param>
+		/// <returns>
+		/// <c>true</c> if <paramref name="value"/> is null or empty; otherwise, <c>false</c>.
+		/// </returns>
+		[Pure]
+		public static bool IsNullOrEmpty([CanBeNull] this StringBuilder value) =>
+			value == null || value.Length == 0;
+
+		/// <summary>
+		/// Determines whether the StringBuilder is netier null nor empty.
+		/// </summary>
+		/// <param name="value">The StringBuilder to test.</param>
+		/// <returns>
+		/// <c>true</c> if <paramref name="value"/> is not null and not empty; otherwise, <c>false</c>.
+		/// </returns>
+		[Pure]
+		public static bool NotNullNorEmpty([CanBeNull] this StringBuilder value) =>
```

Comment:
Maybe `IsNotNullNorEmpty` instead of `NotNullNorEmpty`?

`NotNullNorEmpty` looks like it is used in assertion as so:
```
Assert.NotNullNorEmpty(actual)
```

Reply:
CodeJam uses this convention.

https://github.com/rsdn/CodeJam/commit/4601ec60791ce5e859d528d9115f3aebf30f911f

https://github.com/rsdn/CodeJam/blob/master/CodeJam.Main/Strings/StringExtensions.Infix.cs#L42